llvm/lib/ObjectYAML/ELFYAML.cpp

```diff
@@ -1921,6 +1921,7 @@ void MappingTraits<ELFYAML::VerdefEntry>::mapping(IO &IO,
   IO.mapOptional("Flags", E.Flags);
   IO.mapOptional("VersionNdx", E.VersionNdx);
   IO.mapOptional("Hash", E.Hash);
+  IO.mapOptional("AuxVOffset", E.AuxVOffset);
```

Comment:
If I understand it correctly, all this does is set the value - it doesn't actually cause the data to be written to do a different offset. Is that correct?

Assuming that's the case, I'd suggest naming it after the spec name of `vd_aux`, e.g. `VDAux`. This is to match the similar fields like `SHName` and `SHOffset` for sections, whose only impact is to modify the section header (in contrast with the `Name` and `Offset` fields etc).